src/test/java/com/codeborne/selenide/SelectorsTest.java

```diff
@@ -57,35 +57,35 @@ void byTextEscapesQuotes() {
   void byAttributeUsesXPath() {
     By selector = Selectors.byAttribute("value", "катя");
     assertThat(selector)
-      .hasToString("By.cssSelector: [value='катя']");
+      .hasToString("By.cssSelector: [value=\"катя\"]");
   }
 
   @Test
   void byAttributeEscapesQuotes() {
     By selector = Selectors.byAttribute("value", "Ludvig'van\"Beethoven");
     assertThat(selector)
-      .hasToString("By.cssSelector: [value='Ludvig'van\"Beethoven']");
+      .hasToString("By.cssSelector: [value=\"Ludvig'van\"Beethoven\"]");
```

Comment:
unfortunately this locator is also wrong.

One valid was to test it - run driver.findElement(By.cssSelector(locator)); - it should not throw any exceptions.

Reply:
Actually, I even don't know the working locator for this case. It is simple, when only ' or only " is there, but I don't know the solution if text is  `Ab'cd"de `